### ISSUE TITLE
Sprint 1.27.0 2/2 - June 10 to June 24

### ISSUE THREAD

**Issue by timgl** [PostHog/posthog#10107]
# Global Sprint Planning

## Retro: Status of Outcomes from Previous Sprint

1.
2.
3.
4.
5.

## Retro: What can we do better next sprint?

1.
2.
3.
4.
5.

## Support hero this sprint

Week 1: 
Week 2:

## Plan: Proposed Goals for Next Sprint 

Each goal should have a single owner. Owner can only be an engineer.

Scratch pad for ideas from Tim:
- autocapture/feature flag/experiment support for mobile + Expo support
- 

1.
2.
3.
4.
5.

# Team sprint planning

For your team sprint planning copy this template into a comment below for each team.

```
Team ___

## Retro

<!-- Talk about what went well, what didn't go well and any actions to improve next time -->

- 

## Hang over items from previous sprint

<!-- For each item, decide to re-prioritise (and add below) or deprioritise -->

- Item 1. prioritised/deprioritise

## Planning

<!-- Each item should have a single owner. Owner can only be an engineer. -->

### High priority

-

### Bugs

-

### Low priority / side quests

-

```

**Comment by fuziontech** [comment]
# Team Platform Infrastructure

## Retro

- @fuziontech - Really great sprint. Kafka in AWS and out of :heroku:!!! Made progress on reclaiming disk space on cluster. Gitlab is setup. Still have loads to do, but feeling optimistic about direction. I'm bummed about Bogdan ghosting us, but I'm very happy with the current state of the DE and SRE pipelines. Just have to keep cranking!
- @hazzadous - It was short - Kafka stuff. We got a reasonable amount done. S3 stuff for session recording (Distraction)
- @guidoiaquinti - Pretty hard thb - Starting to struggle juggling all of the workstreams at once. Kafka was a bit thrashy. Support hero is not sustainable. 6 support hero requests just today. Most of these support hero requests are super cursory. We really need to protect engineering time (since it's the most $). Very appreciative of @hazzadous for being a backstop. Very reactive bitnami breaking

## Hang over items from previous sprint

- Having a dev / staging / prod environments setup and running in new AWS account using EKS in parallel with existing ECS cluster. All stateful resources will be shared between the two environments (Kafka, PG, CH, Redis) but **traffic will be able to be consumed from new environment** @guidoiaquinti & @hazzadous (Status: see below)

why: to move towards our 6 month goals of having infrastructure as code

- @fuziontech - Tool to reclaim storage on cluster

Why: to unblock person on events migration

## Planning

### High priority

- Gitlab installation (setup snowplow ingestion plugin and hand holding) @fuziontech 

Why: Support a valuable customer and collect feedback

- Blob storage for session recording setup for a team on cloud @hazzadous 

Why: Reclaim storage on ClickHouse and reduce load on ClickHouse. Reduce costs

- Specifically production traffic will be consumed by new infrastructure in parallel @hazzadous @guidoiaquinti 

Why: Move towards a more repeatable infrastructure defined in code. 6 month goal
